kitsune/gallery/utils.py

```diff
@@ -62,4 +64,5 @@ def check_media_permissions(media, user, perm_type):
 
 
 def get_draft_title(user):
-    return u'Draft for user %s' % user.username
+    return u'Draft for user %s. Created at: %s' % (user.username,
+                                                   datetime.now())
```

Comment:
ah! that makes sense. OK let me just do some quick testing to double check things work.